distribution/ddf-common/src/main/resources/etc/shell.init.script

```diff
@@ -56,6 +56,7 @@ if { %(jlineReader != null) } {
 
   setopt group
   setopt auto-fresh-line
+  unsetopt insert-tab
```

Comment:
Does this conflict with line 78?
https://github.com/codice/ddf/blob/983096dc7cfcc600a34f26780cc4c0ad6edb9e85/distribution/ddf-common/src/main/resources/etc/shell.init.script#L78